### PR TITLE
PCF Kit Update Rel. 24.03

### PR DESCRIPTION
**Pull request by wern**
Contribution from PCF Data usecase.
Updating the existing PCF Exchange Kit to it's latest Catena-X release version (now compatible with Rel. 24.03).

### CODE REVIEW

**Comment by SebastianBezold** [comment]
Hi @wern,

could you please check, that you have an Eclipse Foundation account and signed the Eclipse Contributor Aggreement (ECA)
The email address of your EF account and the one you use for your commits have to match. 
We must not merge this with a failing ECA check

**Comment by wern** [comment]
> Hi @wern,
> 
> could you please check, that you have an Eclipse Foundation account and signed the Eclipse Contributor Aggreement (ECA) The email address of your EF account and the one you use for your commits have to match. We must not merge this with a failing ECA check

Hi @SebastianBezold ,
done. ;-)

**Comment by maximilianong** [comment]
Hello @wern - we tried to build your PR locally.
But we are running into dependency problems - can you have look into this?
Screenshot 2024-02-27 at 7 40 04 PM

File: docs-kits/kits/PCF Exchange Kit/page_adoption-view.md
Comment (maximilianong) [inline docs-kits/kits/PCF Exchange Kit/page_adoption-view.md:1]: Please add a separate section for the standards - like the other KITs are doing it and add the versions e.g. https://eclipse-tractusx.github.io/docs-kits/kits/Business%20Partner%20Kit/Adoption%20View#standards

**Review by maximilianong — CHANGES_REQUESTED** [review]

File: docs-kits/kits/PCF Exchange Kit/changelog.md [hunk 1]
```diff
@@ -8,6 +8,24 @@ sidebar_position: 1
 
 All notable changes to this Kit will be documented in this file.
 
+## [1.0.0] - 2024-02-12
```
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply docs-kits/kits/PCF Exchange Kit/changelog.md]: Changed as requested, see [b7555db](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/710/commits/b7555dbaebf0d2f11cc0c26a733514998c50b897)

File: docs-kits/kits/PCF Exchange Kit/page_adoption-view.md
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply docs-kits/kits/PCF Exchange Kit/page_adoption-view.md:1]: Changed as requested, see [287c8e5](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/710/commits/287c8e524673c9320d4bb4ec32e2dc7507d3e73d)

Comment (SMaierTSI) [reply docs-kits/kits/PCF Exchange Kit/page_adoption-view.md:1]: Adapted and added standard section as requested.

File: docs-kits/kits/PCF Exchange Kit/Software Development View/page_software-development-view.md
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply docs-kits/kits/PCF Exchange Kit/Software Development View/page_software-development-view.md:1]: Empty operation view page deleted as requested, see [a8feb1d](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/710/commits/a8feb1df39a607b400baa354fa4384b0fa212916)

File: docs-kits/kits/PCF Exchange Kit/page_adoption-view.md
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply docs-kits/kits/PCF Exchange Kit/page_adoption-view.md:1]: Adapted in notice section as requested, see [9aec609](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/710/commits/9aec60946bbde769ed7d24230482d036c97b5313)

File: docs-kits/kits/PCF Exchange Kit/Software Development View/page_software-development-view.md [hunk 2]
```diff
@@ -200,31 +208,31 @@ For more examples how to define policies with SSI have a look [here](https://git
 
 ## Error Handling
 
-As Release 3.2 only covers the "happy path" for exchange PCF data via the Catena-X network. Error handling is currently not covered.
+In case no material twin or no PCF submodel is found, EDC Asset type has to be used to find the Endpoint needed to perform the PCF request.
 
 ## Standards
 
 ### Used CX Standards
```
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply docs-kits/kits/PCF Exchange Kit/Software Development View/page_software-development-view.md:215]: @maximilianong : after discussion in our PCF use case team we would leave the standards as specified in the software development view as they represent the technical details which are probably useful for a technical audience. In our adoption view, we focus on business process/ methodological aspects, so there we only list the according general standards e.g. PCF rulebook. Would it be ok if we don't adapt this as requested by you?

File: docs-kits/kits/PCF Exchange Kit/Software Development View/page_software-development-view.md [hunk 3]
```diff
@@ -200,31 +208,31 @@ For more examples how to define policies with SSI have a look [here](https://git
...
+- [CX-0049 DID Document Schema v1.0.0](https://catena-x.net/de/standard-library)
+- [CX-0050 Framework Agreement Credential v1.0.0](https://catena-x.net/de/standard-library)
+- [CX-0051 Summary Credential v1.0.0](https://catena-x.net/de/standard-library)
 
 ## Other Standards
 
-- [Pathfinder Framework](https://wbcsd.github.io/data-exchange-protocol/v2/)
+- [Pathfinder Framework - v2.0.0](https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221)
 
 ## NOTICE
```
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply docs-kits/kits/PCF Exchange Kit/Software Development View/page_software-development-view.md:237]: Adapted as requested, see [fb1d856](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/710/commits/fb1d856d36bc0eb4225518ea212fee0720ff367f)

File: docs-kits/kits/PCF Exchange Kit/Software Development View/pcf-exchange-api/get-pcf.api.mdx
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply docs-kits/kits/PCF Exchange Kit/Software Development View/pcf-exchange-api/get-pcf.api.mdx:1]: @maximilianong : can you please tell us if we can achieve this by just deleting the docs-kits/kits/PCF Exchange Kit/Software Development View/pcf-exchange-api/catena-x-pcf-request-endpoint.info.mdx file? If so i would delete the according file

**Comment by danielmiehle** [comment]
@SMaierTSI thank you for fixing the change requests. Can you also have a look at the current linting errors, please?

image

File: docs-kits/kits/PCF Exchange Kit/Software Development View/pcf-exchange-api/get-pcf.api.mdx
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply docs-kits/kits/PCF Exchange Kit/Software Development View/pcf-exchange-api/get-pcf.api.mdx:1]: Thank you @SujitMBRDI , guess we will pick the 2nd option

**Comment by SMaierTSI** [comment]
@danielmiehle : fixed the linting errors for the PCF Exchange KIT. However, there's still a large number of errors referring to other KITS which are not in scope of PCF Exchange

File: docs-kits/kits/PCF Exchange Kit/Software Development View/page_software-development-view.md
[the remark this answers is not in the document]
Comment (wern) [reply docs-kits/kits/PCF Exchange Kit/Software Development View/page_software-development-view.md:1]: @maximilianong after deleting `operating_view.md` Docusaurus refuses to start:

> These sidebar document ids do not exist:
> - kits/PCF Exchange Kit/Operation View

Do we want to change sidebar structure here or just add `operating_view.md` with a note "No open source software available at the moment." ?

File: docs-kits/kits/PCF Exchange Kit/Software Development View/pcf-exchange-api/get-pcf.api.mdx
[the remark this answers is not in the document]
Comment (wern) [reply docs-kits/kits/PCF Exchange Kit/Software Development View/pcf-exchange-api/get-pcf.api.mdx:1]: @maximilianong deleting the page is also not an option here, as it looks like Docusaurus structure will get messed up when doing so. So either we change sidebar structure here or add some info to the page like: 
"PCF Exchange API to be used to request, return or update PCF data via the Catena-X network."

**Review by danielmiehle — DISMISSED** [review]

**Review by maximilianong — DISMISSED** [review]

**Review by maximilianong — APPROVED** [review]

**Review by danielmiehle — APPROVED** [review]

**Pull request merged**